src/textual/command.py

```diff
@@ -891,7 +891,15 @@ def _refresh_command_list(
             if command_list.highlighted is not None and not clear_current
             else None
         )
-        command_list.clear_options().add_options(commands)
+
+        def sort_key(command: Command) -> float:
```

Comment:
Could you add a constant / property on DiscoveryHit that is always 0? The feels a little cleaner than the getattr to me.

I'm actually somewhat surprised that one doesn't inherit from the other. I wonder what the thinking was behind that?

Reply:
I think because DiscoveryHits don't have a "score", and they were also added later. I can understand why someone might want to not give DiscoveryHits a score hard-coded to 0 from a design POV. That's why I chose the getattr too, but it doesn't particularly matter that much so I'll change it on Monday.